ruff.toml

```diff
@@ -12,8 +12,32 @@ select = [
 	"ASYNC",  # Flake8 async
 	"ISC",  # flake8-implicit-str-concat
     "TID",  # flake8-tidy-imports
+    "T10", # Debug statements
```

Comment:
are we mixing tabs and spaces i this file?

Reply:
weird that didn't show up locally, but I've fixed it.